command/exec.go

```diff
@@ -0,0 +1,53 @@
+package command
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/wantedly/developers-account-mapper/store"
+)
+
+type ExecCommand struct {
+	Meta
+}
+
+func (c *ExecCommand) Run(args []string) int {
+	loginName := args[0]
+
+	s := store.NewDynamoDB()
+
+	user, err := s.GetUserByLoginName(loginName)
+	if err != nil {
+		log.Println(err)
+		return 1
+	}
+
+	envs := os.Environ()
+	envs = append(envs, fmt.Sprintf("%s=%s", "GITHUB_USERNAME", user.GitHubUsername))
+
+	execCmd := exec.Command(args[1], args[2:]...)
+	execCmd.Env = envs
+	execCmd.Stderr = os.Stderr
+	execCmd.Stdout = os.Stdout
+	err = execCmd.Run()
+	if err != nil {
+		log.Println(err)
+		return 1
```

Comment:
@dtan4 
Implemented as instructed, but I can not fetch subcommand exit status.

```console
potsbo@magi-INS-% envchain wtd bin/developers-account-mapper exec potsbo foo
2017/01/11 10:45:38 exec: "foo": executable file not found in $PATH
potsbo@magi-INS-% echo $?
1
potsbo@magi-INS-% foo
zsh: command not found: foo
potsbo@magi-INS-% echo $?
127
```

Before I look into the detail of go and system call, I want your opinion, am I doing anything wrong or this is how it is?